pkgs/by-name/no/notonoto/package.nix

```
@@ -14,10 +17,29 @@ stdenvNoCC.mkDerivation rec {
     hash = "sha256-1dbx4yC8gL41OEAE/LNDyoDb4xhAwV5h8oRmdlPULUo=";
   };
 
+  nativeBuildInputs = [
+    fontforge
+    python3Packages.setuptools
+  ];
+
+  buildInputs = with python3Packages; [
+    fonttools
+    ttfautohint-py
+  ];
+
+  buildPhase = ''
+    runHook preBuild
+
+    fontforge --script fontforge_script.py
+    fontforge --script fonttools_script.py
```

This script should be executed with python and not with fontforge.
```suggestion
    python ./fonttools_script.py
```